manifests/params.pp

```diff
@@ -23,15 +23,17 @@
   $show_diff           = false
   $module_repository   = undef
   if versioncmp($::puppetversion, '4.0') < 0 {
-    $configtimeout           = 120
-    $server_puppetserver_dir = '/etc/puppetserver'
-    $server_ruby_load_paths  = []
-    $server_jruby_gem_home   = '/var/lib/puppet/jruby-gems'
+    $configtimeout              = 120
+    $server_puppetserver_dir    = '/etc/puppetserver'
+    $server_puppetserver_vardir = undef
```

Comment:
I think we also support puppetserver 1 which is puppet 3. Won't this break and should it be ```$vardir```?